src/App.tsx

```diff
@@ -14,11 +16,95 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGoods: string[];
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    selectedGoods: ['Jam'],
+  };
+
+  remove = (good: string) => {
+    this.setState((state) => ({
+      selectedGoods: state.selectedGoods.filter(item => item !== good),
+    }));
+  };
+
+  add = (good: string) => {
+    this.setState((state) => ({
+      selectedGoods: [...state.selectedGoods, good],
+    }));
+  };
+
+  addOrDelGood = (toggler: boolean, good: string) => {
+    if (toggler) {
+      this.remove(good);
+    } else {
+      this.add(good);
+    }
+  };
+
+  amountGoods = (amount: number) => {
```

Comment:
```suggestion
  formatTitle = (amount: number) => {
```
or something like that